test/xpu/test_spectral_ops_xpu.py

```diff
@@ -0,0 +1,81 @@
+# Owner(s): ["module: intel"]
+
+import torch
+import numpy as np
+from packaging import version
+from itertools import product
+
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests, ops, onlyNativeDeviceTypes)
+from torch.testing._internal.common_methods_invocations import (
+    spectral_funcs, SpectralFuncType)
+from torch.testing._internal.common_utils import run_tests
+
+try:
+    from .xpu_test_utils import XPUPatchForImport
+except Exception as e:
+    from ..xpu_test_utils import XPUPatchForImport
+
+with XPUPatchForImport(False):
+    from test_spectral_ops import TestFFT
+
+has_scipy_fft = False
+try:
+    import scipy.fft
+    has_scipy_fft = True
+except ModuleNotFoundError:
+    pass
+
+REFERENCE_NORM_MODES = (
+    (None, "forward", "backward", "ortho")
+    if version.parse(np.__version__) >= version.parse('1.20.0') and (
+        not has_scipy_fft or version.parse(scipy.__version__) >= version.parse('1.6.0'))
+    else (None, "ortho"))
+
+@ops([op for op in spectral_funcs if op.ndimensional == SpectralFuncType.OneD],
+     allowed_dtypes=(torch.float, torch.cfloat))
+def _test_reference_1d(self, device, dtype, op):
```

Comment:
Do you mean we only support this case in test_spectral? or we only override this case?

Reply:
We only override `_test_reference_1d` to modify `atol` and `rtol`.